.github/workflows/container-build-feature-repo.yml

```diff
@@ -43,10 +43,34 @@ on:
         default: ${{ github.event.repository.name }}
         description: The repository that triggered the workflow to better tag the image
 
-
 jobs:
+  style:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install linters
+        run: sudo apt-get install -y libxml2-utils uncrustify black
+
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10" # same as Ubuntu22.04
```

Comment:
:thinking: I though about that, and I forgot :) I guess with this one doesn't need to also manually pull the binary depdencies, I will give it a try

Reply:
@nachovizzo I actually don't remember anymore but saw that they install colcon stuff in the [`ros-base`](https://github.com/osrf/docker_images/blob/bca53bf4c09d771be3ff735da4157203b53ebc2b/ros/iron/ubuntu/jammy/ros-base/Dockerfile#L9) they install colcon extension. It might be even in [`ros-core`](https://github.com/osrf/docker_images/tree/bca53bf4c09d771be3ff735da4157203b53ebc2b/ros/iron/ubuntu/jammy/ros-core) :shrug:

Reply:
it's on ros-base :)